cpp/net/url_fetcher.cc

```diff
@@ -40,9 +42,13 @@ struct State {
     CHECK(!conn_) << "request state object still had a connection at cleanup?";
   }
 
+  // The following methods are only called on the libevent dispatch
```

Comment:
Also, there's `Base::OnEventThread()` if you want to ensure it's so (or just log if it isn't).